docs/conf.py

```diff
@@ -0,0 +1,124 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
+
+"""Sphinx documentation builder."""
+
+# -- General configuration ---------------------------------------------------
+import datetime
+import doctest
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+sys.path.append(os.path.abspath("."))
+
+project = "Qiskit Algorithms"
+copyright = f"2017-{datetime.date.today().year}, Qiskit Algorithms Development Team" # pylint: disable=redefined-builtin
+author = "Qiskit Algorithms Development Team"
+
+# The short X.Y version
+version = "0.1"
+# The full version, including alpha/beta/rc tags
+release = "0.1.0"
+
+extensions = [
+    "sphinx.ext.napoleon",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.mathjax",
+    "sphinx.ext.viewcode",
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.doctest",
+    "reno.sphinxext",
+    "sphinx_design",
+    "matplotlib.sphinxext.plot_directive",
+    "sphinx.ext.doctest"
+]
+
+templates_path = ["_templates"]
+
+# Number figures, tables and code-blocks if they have a caption.
+numfig = True
+# Available keys are 'figure', 'table', 'code-block' and 'section'.  '%s' is the number.
+numfig_format = {"table": "Table %s"}
+
+# The language for content autogenerated by Sphinx or the default for gettext content translation.
+language = "en"
+
+# Relative to source directory, affects general discovery, and html_static_path and html_extra_path.
+exclude_patterns = ["_build", "**.ipynb_checkpoints"]
+
+pygments_style = "colorful"
+
+# Whether module names are included in crossrefs of functions, classes, etc.
+add_module_names = False
+
+# A list of prefixes that are ignored for sorting the Python module index
+# (e.g., if this is set to ['foo.'], then foo.bar is shown under B, not F).
+modindex_common_prefix = ["qiskit."]
+
+intersphinx_mapping = {
+    "rustworkx": ("https://qiskit.org/ecosystem/rustworkx/", None),
+    "qiskit-ibm-runtime": ("https://qiskit.org/ecosystem/ibm-runtime/", None),
+    "qiskit-aer": ("https://qiskit.org/ecosystem/aer/", None),
+    "numpy": ("https://numpy.org/doc/stable/", None),
+    "matplotlib": ("https://matplotlib.org/stable/", None),
+}
+
+# -- Options for HTML output -------------------------------------------------
+
+html_theme = "qiskit_sphinx_theme"
+html_last_updated_fmt = "%Y/%m/%d"
+html_theme_options = {
+    "logo_only": True,
+    "display_version": True,
+    "prev_next_buttons_location": "bottom",
+    "style_external_links": True,
+}
```

Comment:
```suggestion
html_theme = "qiskit-ecosystem"
html_last_updated_fmt = "%Y/%m/%d"
html_title = f"{project} {release}"
```

This will use the much nicer Qiskit Ecosystem theme. `html_title` will make the project name in the left sidebar less noisy.